Makefile

```diff
@@ -21,11 +21,15 @@ minigeth_default_arch:
 mipsevm:
 	cd mipsevm && go build
 
-contracts: yarn
+contracts: nodejs
 	npx hardhat compile
 
-yarn:
-	yarn install
```

Comment:
Seems like there is no particular benefit from it — besides [faster install time](https://pnpm.io/benchmarks), so I thought I'd cut one dependency (since you need npm to have yarn anyway).

I'd like to encourage people to use `pnpm` if they want speedy installation. It seems faster than even yarn in most cases (cf. benchmarks above) and deduplicates storage accross different projects and even different versions of the same dependency. Given that depending on hardhat, ethers, etc pulls in 500MB or more, not having every solidity project pull in an extra 500MB seems like a big win. That's why the `nodejs` command uses `pnpm` if installed.

If you're really passionate about yarn, we could always add another clause in the `if` statement in the `nodejs` so that it uses `yarn` if avaible and `pnpm` is not available.